Parser/Controllers/VisitorsController.cs

```diff
@@ -0,0 +1,106 @@
+﻿using MongoDB.Driver;
+using System.Web.Http;
+using Visitors.Models;
+using System.Threading.Tasks;
+using MongoDB.Bson;
+
+namespace Visitors.Controllers
+{
+    public class VisitorsController : ApiController
+    {
+        private IMongoCollection<VisitPage> CollectionPages { get; set; }
+        private IMongoCollection<Visit> CollectionVisits { get; set; }
+        private IMongoCollection<Visitor> CollectionVisitors { get; set; }
+
+        public VisitorsController()
+        {
+            var client = new MongoClient();
+            var database = client.GetDatabase("populator");
+            CollectionPages = database.GetCollection<VisitPage>("sp_visit_pages");
+            CollectionVisits = database.GetCollection<Visit>("sp_visits");
+            CollectionVisitors = database.GetCollection<Visitor>("sp_visitors");
+        }
+
+
+        // GET: api/Visitors/5
+        public string Get(int id)
+        {
+            return "value";
+        }
+
+        // PATCH: api/Visitors
+        public async Task Patch()
+        {
+
+            var filter = new BsonDocument();
+            var findOptions = new FindOptions { NoCursorTimeout = true };
+
+            #region sp_visitors
+            int count = 0;
+            var builderVisitors = Builders<Visitor>.Filter;
+            var updateVisitors = Builders<Visitor>.Update;
+
+            using (var cursorVisitors = await CollectionVisitors.Find(filter, findOptions).ToCursorAsync())
+            {
+                while (await cursorVisitors.MoveNextAsync())
+                {
+                    var dateVisitors = cursorVisitors.Current;
+                    if (count >= 1000) break;
+
+                    foreach (Visitor docVisitor in dateVisitors)
+                    {
+                        
+                        var filterVisitors = builderVisitors.Eq("UniqueID", docVisitor.UniqueID);
+
+                        //visits->visitors
+                        var builderVisits = Builders<Visit>.Filter;
+                        var filterVisits = builderVisits.Eq("OwnerID", docVisitor.UniqueID);
+                        var resultVisits = await CollectionVisits.Find(filterVisits).ToListAsync();
```

Comment:
use Lambda expressions where applicable instead of filtering with string value of Field "OwnerID"
await CollectionVisits.Find(v => v.OwnerID == docVisitor.UniqueID)